Sources/SATSCore/Assets/ThemeColors.xcassets/Silver Theme/Background Top/silverGradientStart.colorset/Contents.json

```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0.455",
-          "green" : "0.416",
-          "red" : "0.376"
+          "blue" : "0x74",
```

Comment:
Is this color correct? I find 14233C for Gradient Silver Start (light mode).

Reply:
same here

Reply:
Ok, can confirm this is correct.